rtrlib/pfx/trie/trie-pfx.c

```diff
@@ -133,18 +153,31 @@ inline struct trie_node *pfx_table_get_root(const struct pfx_table *pfx_table, c
 
 int pfx_table_del_elem(struct node_data *data, const unsigned int index)
 {
+    struct data_elem *tmp;
+    struct data_elem deleted_elem = data->ary[index];
```

Comment:
Yes, I totally agree that asserts() can be very useful when debugging to find the source of an error.